src/Arriba/Arriba/Diagnostics/Tracing/ArribaEventSource.cs

```diff
@@ -31,11 +32,15 @@ public interface IEventSource
         EventListener EnableEvents(EventListener listener, EventLevel level);
     }
 
-    public interface IArribaEvents : IEventSource
+    public interface IArribaServiceEvents
     {
-        void ServiceStart<T>();
-        void ServiceComplete<T>();
+        void Starting<T>();
+        void Completing<T>();
         void TrackFatalException(Exception ex, IServiceIdentity id);
+    }
+
+    public interface IArribaEvents : IEventSource
```

Comment:
Discussion.  Size of this interface is getting large.  Have we thought of ways to group these in more granular interfaces as needed by the relevant services?  In my head they may be all present on the concrete EventSource derived class, but smaller ones will be desired for testing (of services and of our logging).